src/SonarScanner.MSBuild.Common/SingleGlobalInstanceMutex.cs

```diff
@@ -58,7 +58,7 @@ public void Dispose()
 
         private static Mutex CreateMutex(string name)
         {
-#if NET46
+#if NET462
```

Comment:
Looking at the [Applies to section](https://learn.microsoft.com/en-us/dotnet/api/system.security.accesscontrol.mutexsecurity?view=dotnet-plat-ext-7.0#applies-to) of `MutexSecurity`, it seems more use full to check for `#if !NETSTANDARD` or `#if NETFRAMEWORK` as it is available everywhere. It just happens to be not part of .Net Standard.